data/snippets.ts

```diff
@@ -1099,6 +1099,20 @@ const misc: Snippet[] = [
     keyword: "standup",
     type: "template",
   },
+  {
+    id: nanoid(),
+    name: "LinkedIn Profile URL",
+    text: "https://linkedin.com/in/username",
+    keyword: "in",
+    type: "template",
+  },
+  {
+    id: nanoid(),
+    name: "Twitter / X Profile URL",
+    text: "https://x.com/username",
```

Comment:
should we make use of the `{cursor}` placeholder here?

Reply:
I’d say no, it works similarly to `!cal`